android/quest/src/main/java/org/smartregister/fhircore/quest/QuestApplication.kt

```diff
@@ -43,51 +35,11 @@ class QuestApplication : Application(), DataCaptureConfig.Provider {
     if (BuildConfig.DEBUG) {
       Timber.plant(Timber.DebugTree())
     }
-
-    configureSingletonDefaultValidationSupport()
   }
 
   override fun getDataCaptureConfig(): DataCaptureConfig {
     configuration =
       configuration ?: DataCaptureConfig(attachmentResolver = referenceAttachmentResolver)
     return configuration as DataCaptureConfig
   }
-
-  // TODO https://github.com/google/android-fhir/issues/1173
-  // Action: Remove once fixed
-  // Once SDK resolves the issue this can be removed from here as there would be no
-  // duplication of objects
-  private fun configureSingletonDefaultValidationSupport() {
-
-    CoroutineScope(Dispatchers.Default).launch {
-      val fhirContext =
-        FhirContext.forR4Cached().apply {
-          Timber.i("Loading FhirContext.forR4Cached on application init")
-        }
-      ResourceMapper.run {
-        val validationSupport = extractResourceMapperValidationSupport()
-        fhirContext.validationSupport = validationSupport
-        Timber.i("Loading ResourceMapper on application init")
-      }
-    }
-  }
-
-  // TODO https://github.com/google/android-fhir/issues/1173
-  // Action: Remove once fixed
-  private fun extractResourceMapperValidationSupport() =
-    ResourceMapper::class
-      .java
-      .getDeclaredField("fhirPathEngine")
-      .also { fhirPathEngineProperty -> fhirPathEngineProperty.isAccessible = true }
-      .get(null)
-      .let { fhirPathEngine ->
-        ((fhirPathEngine as FHIRPathEngine).worker as HapiWorkerContext).let { hapiWorkerContext ->
-          hapiWorkerContext
-            .javaClass
-            .getDeclaredField("myValidationSupport")
-            .also { myValidationSupportProperty -> myValidationSupportProperty.isAccessible = true }
-            .get(hapiWorkerContext) as
-            DefaultProfileValidationSupport
-        }
-      }
 }
```

Comment:
Removed these, as it seems the v0.1.0-beta03-preview5-SNAPSHOT of datacapture has changes from https://github.com/google/android-fhir/pull/1380